commit 42a75f1a3f0b3c9f1a297476de39f95ec8933ed2 (refs/changes/58/458/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-14T00:14:08+00:00 (3 years, 1 month ago)
    
    CNS-158 lock sdc-scripts dep to a specific git sha
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/package.json b/package.json
index 9a966f1..9c00737 100644
--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "restify": "4.0.3",
     "restify-clients": "1.1.1",
     "restify-errors": "4.2.0",
-    "sdc-scripts": "git+https://github.com/joyent/sdc-scripts.git",
+    "sdc-scripts": "git+https://github.com/joyent/sdc-scripts.git#d01c308",
     "sprintf-js": "^1.0.3",
     "triton-tags": "^1.1.2",
     "uuid": "2.0.1",
